docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -458,6 +459,15 @@ the property may be configured for:
        retry a request.
      - ``10``
      - Azure Blob Storage
+   * - ``exchange.hdfs.block-size``
+     - Block size for HDFS storage.
+     - ``4MB``
+     - HDFS
+   * - ``hdfs.config.resources``
+     - Comma-separated list of HDFS configuration files. These files must
```

Comment:
its a list of paths to the conf files.. 

Comma-separated list of paths to HDFS configuration files, for example ``/etc/hdfs-site.xml`` The files must exist on all nodes in the Trino cluster.


Related question.. are relative paths allowed? If yes.. what are they relative to?

Reply:
Updated the sentence.
Only absolute paths will work.